### DISCUSSION TITLE
Where can I find the log file when using the HA add-on?

### DISCUSSION THREAD

**Discussion by juskalalie** [wmbusmeters/wmbusmeters#739] [Q&A] [answered]
Hello, total n00b here...
I see in the config that the log is saved in /dev/stdout, so I understand this is only the log output from the add-on page.
Is there another log file created somewhere?
Thanks a lot!

**Comment by BIBOLV** [comment, marked as answer]
Hi, log files are being sent to HA so you can get them in Settings > System > Logs and then choose wmbusmeters on top, or look in other sections for what you need.

**Reply by juskalalie** [reply]
Thank you very much! Is there a way to have the timestamps shown in the logs? It would be useful to know what time a telegram was sent.
Thanks!

**Reply by BIBOLV** [reply]
Try `logtimestamps=always`, not sure if it will work as expected for HA addon though.

**Reply by juskalalie** [reply]
That worked, thank you very much!
